resources/web/wwi/FloatingRobotWindow.js

```diff
@@ -0,0 +1,131 @@
+export default class FloatingRobotWindow {
+    constructor(parentNode, name, window, url) {
+        this.name = name;
+        this.window = window;
+        this.url = url;
+
+        this.floatingRobotWindow = document.createElement('div');
+        this.floatingRobotWindow.className = 'floating-robotWindow';
+        this.floatingRobotWindow.id = name;
+        this.floatingRobotWindow.style.visibility = 'hidden';
+        parentNode.appendChild(this.floatingRobotWindow);
+        
+        this.floatingRobotWindowHeader = document.createElement('div');
+        this.floatingRobotWindowHeader.className = 'floating-robotWindow-header';
+        this.floatingRobotWindow.appendChild(this.floatingRobotWindowHeader);
+
+        this.headerText = document.createElement('p');
+        this.headerText.className = 'floating-robotWindow-text';
+        this.headerText.innerHTML = 'Robot Window: ' + name;
+        this.floatingRobotWindowHeader.appendChild(this.headerText);
+
+        this.headerQuit = document.createElement('button');
+        this.headerQuit.className = 'floating-robotWindow-close';
+        this.headerQuit.id = 'close-'+name;
+        this.headerQuit.innerHTML = ('&times');
+        this.floatingRobotWindowHeader.appendChild(this.headerQuit);
+
+        this.floatingRobotWindowContent = document.createElement('div');
+        this.floatingRobotWindowContent.className = 'floating-robotWindow-content';
+        this.floatingRobotWindow.appendChild(this.floatingRobotWindowContent);
+
+        let robotWindow = document.createElement('iframe');
+        robotWindow.id = this.name+"-robotWindow";
+        robotWindow.src = this.url+"/robot_windows/"+this.window+"/"+this.window+".html?name="+this.name;
```

Comment:
Coding style: there should be spaces around the `+` and `?` operators.
This happens at other places in this code.